spec/acceptance/api/backend_api_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+resource 'BackendApi' do
+  let(:resource) { FactoryBot.create(:backend_api) }
+  let(:expected_properties) { %w[id name system_name description private_endpoint account_id created_at updated_at] }
+
+  json(:resource) do
+    let(:root) { 'backend_api' }
+
+    it { subject.should have_properties(expected_properties).from(resource) }
+  end
+
+  json(:collection) do
+    let(:root) { 'backend_apis' }
+    context do
+      let(:collection) { [resource, resource] }
+      it 'contains the backend apis data by its representer' do
+        subject.each do |subject_backend_api|
```

Comment:
But they are no different so if one pass the other too...

Reply:
Fixed 👍